File: planningMaintenance/src/main/java/net/elghz/planningmaintenance/model/Site.java
package net.elghz.planningmaintenance.model;

import lombok.Data;

@Data
public class Site {

    private Long id;
    private String code;
    private String typeSite;
    private String name;
    private Double latitude;
    private Double longitude;
    private String addresse;
    private String typeInstallation;
    private String typeAlimentation;
    private String typeTransmission;
    private Boolean presenceGESecours;
    private String supportAntennes;
    private Double hauteurSupportAntenne;
    private String lieuInsatallationBTS;

}

File: frontEnd-Angular/src/app/models/Site.ts
import {CentreTechnique} from "./centreTechnique";
import {Equipement} from "./equipement";
import {Photo} from "./photo";
import {Immuble} from "./immuble";

export interface Site{
   id:number
    code:String;
   name:string;
   typeSite :String;
  latitude:number;
   longitude:number;
   addresse:String;
   typeInstallation :String;
  typeAlimentation:String;
 typeTransmission:String;
  presenceGESecours:String;
  lieuInsatallationBTS:String;
  hauteurSupportAntenne:String;
  supportAntennes:String
  centreTechnique:CentreTechnique
  equipements:Equipement[];
  photos: Photo[];
  getPhotosImagePaths(): string[];

}
